src/net/SDP/SDPMessageMediaFormat.cs

```diff
@@ -0,0 +1,31 @@
+﻿//-----------------------------------------------------------------------------
+// Filename: SDPMessageMediaFormat.cs
+//
+// Description: Contains enums and helper classes for common definitions
+// and attributes used in SDP payloads.
+//
+// Author(s):
+// Jacek Dzija
+// Mateusz Greczek
+//
+// History:
+// 30 Mar 2021 Jacek Dzija,Mateusz Greczek Added MSRP
+//
+// License: 
+// BSD 3-Clause "New" or "Revised" License, see included LICENSE.md file.
+//-----------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace SIPSorcery.net.SDP
+{
```

Comment:
Could the namespace be changed to SIPSorcery.Net so that it's consistent with the other SDP related classes.

Reply:
changed

Reply:
> Could you add some unit tests with examples of parsing an MSRP SDP payload? That would make it a lot clearer what the PR is doing.

added UTs